### DISCUSSION TITLE
Charging limit not working properly

### DISCUSSION THREAD

**Discussion by Albert24GG** [seerge/g-helper#1402] [General]
I just noticed that if the battery is charged to a higher level than the charging limit, it will begin to consume the battery until it reaches the limit. For example, my battery level is 80%, the charging limit is 60%, it starts to drain the battery. To me, this doesn't seems right since it adds unnecessary wear. Isn't it possible to adopt the implementation used by asusctl to bypass completely the battery (and use just the barrel charger) when the charging limit is lower than the actual battery level?

**Comment by seerge** [comment]
@Albert24GG hello, i personally don't experience this. It depends on a hardware implementation of the charging on your device.

The only thing you can do via software - is to set a value of when to stop charging :) This is charging limit, and it's already working.

**Reply by Albert24GG** [reply]
Yeah, it might have been a glitch or something cause now it seems to be working properly. Thank you for the response!
